source/tyr/tyr/resources.py

```diff
@@ -1113,3 +1116,28 @@ def delete(self, name=None):
             raise
         return ({}, 204)
 
+class Data(flask_restful.Resource):
+    def post(self, instance_name):
```

Comment:
In the first place, we wanted to let the directory scan create the job. Is there a problem with that ?